src/components/Datagrid/components/TooltipContainer.tsx

```diff
@@ -48,9 +47,14 @@ export const TooltipContainer: React.FC<TooltipContainerProps> = ({
         __left={x + width / 2}
         zIndex="2"
       >
-        {/* TODO: use tooltip from new macaw when it will be ready */}
-        <Tooltip open={true} title={title} placement="top">
-          <span></span>
+        <Tooltip open={true}>
```

Comment:
**question**: does it mean that Tooltip will always be open?

Reply:
It mean that datagrid control hover not Tooltip itself so when hover happened I render tooltip already open